### PR TITLE
Add the missing get_identity prefix to google_analytics_gtag, test it

### PR DESCRIPTION
**Pull request by PetrDlouhy**
Add the missing prefix in `get_identity()` in `google_analytics_gtag` and test it.

Change of docs and other tests will come in separate PRs.

### CODE REVIEW

File: tests/unit/test_tag_google_analytics_gtag.py [hunk 1]
```diff
@@ -59,6 +59,21 @@ def test_identify(self):
...
+            'analytical_identity': 'bar',
+            'user': User(username='test'),
+        }))
+        assert "gtag('set', {'user_id': 'foo'});" in r
+
+        r = GoogleAnalyticsGTagNode().render(Context({
+            'analytical_identity': 'bar',
+            'user': User(username='test'),
+        }))
+        assert "gtag('set', {'user_id': 'bar'});" in r
```
Comment (bittner) [inline tests/unit/test_tag_google_analytics_gtag.py:75]: Can you split this up into two separate tests? IIUC, the two tests don't build on each other.

File: tests/unit/test_tag_google_analytics_gtag.py [hunk 2]
```diff
@@ -59,6 +59,21 @@ def test_identify(self):
         r = GoogleAnalyticsGTagNode().render(Context({'user': User(username='test')}))
         assert "gtag('set', {'user_id': 'test'});" in r
 
+    def test_identity_context(self):
+        """ Test when identity is set in the context """
```
Comment (bittner) [inline tests/unit/test_tag_google_analytics_gtag.py:63]: Don't use spaces at the beginning and the end of the docsstring.

Don't mention the obvious ("Test when"), get straight to the point instead, e.g. 

```
        """Identity must be set in the context when ..."""
```

**Review by bittner — CHANGES_REQUESTED** [review]

File: tests/unit/test_tag_google_analytics_gtag.py (hunk 1 above)
Comment (PetrDlouhy) [reply tests/unit/test_tag_google_analytics_gtag.py]: OK

File: tests/unit/test_tag_google_analytics_gtag.py (hunk 2 above)
Comment (PetrDlouhy) [reply tests/unit/test_tag_google_analytics_gtag.py]: OK. The docstring is now too long to be one-liner, so it has to be split anyway.

**Comment by PetrDlouhy** [comment]
Hope I fixed all remarks.

File: tests/unit/test_tag_google_analytics_gtag.py [hunk 3]
```diff
@@ -59,6 +59,28 @@ def test_identify(self):
...
         assert "gtag('set', {'user_id': 'test'});" in r
 
+    def test_identity_context_specific_provider(self):
+        """
+        The user_id variable must be set according to
+        google_analytics_gtag_identity variable in the context.
+        """
+        r = GoogleAnalyticsGTagNode().render(Context({
+            'google_analytics_gtag_identity': 'foo_gtag_identity',
+            'analytical_identity': 'bar_analyitical_identity',
```
Comment (bittner) [inline tests/unit/test_tag_google_analytics_gtag.py:69]: I understand this is a test and the values can be arbitrary. Yet still, the typos are not intentional, are they?

```suggestion
            'analytical_identity': 'bar_analytical_identity',
```

File: tests/unit/test_tag_google_analytics_gtag.py [hunk 4]
```diff
@@ -59,6 +59,28 @@ def test_identify(self):
...
+
+    def test_identity_context_general(self):
+        """
+        The user_id variable must be set according to analyitical_identity variable in the context.
+        """
+        r = GoogleAnalyticsGTagNode().render(Context({
+            'analytical_identity': 'bar_analyitical_identity',
+            'user': User(username='test'),
+        }))
+        assert "gtag('set', {'user_id': 'bar_analyitical_identity'});" in r
```
Comment (bittner) [inline tests/unit/test_tag_google_analytics_gtag.py:82]: Same here:

```suggestion
            'analytical_identity': 'bar_analytical_identity',
            'user': User(username='test'),
        }))
        assert "gtag('set', {'user_id': 'bar_analytical_identity'});" in r
```

**Review by bittner — CHANGES_REQUESTED** [review]

**Comment by PetrDlouhy** [comment]
@bittner I fixed the typos.

File: tests/unit/test_tag_google_analytics_gtag.py [hunk 5]
```diff
@@ -59,6 +59,28 @@ def test_identify(self):
...
+        r = GoogleAnalyticsGTagNode().render(Context({
+            'google_analytics_gtag_identity': 'foo_gtag_identity',
+            'analytical_identity': 'bar_analytical_identity',
+            'user': User(username='test'),
+        }))
+        assert "gtag('set', {'user_id': 'foo_gtag_identity'});" in r
+
+    def test_identity_context_general(self):
+        """
+        The user_id variable must be set according to analyitical_identity variable in the context.
```
Comment (bittner) [inline tests/unit/test_tag_google_analytics_gtag.py:76]:
```suggestion
        The user_id variable must be set according to analytical_identity variable in the context.
```

**Review by bittner — CHANGES_REQUESTED** [review]
Found another typo, sorry! :worried:

**Review by bittner — APPROVED** [review]

**Comment by bittner** [comment]
@PetrDlouhy Thanks for improving django-analytical! :1st_place_medal:

**Pull request merged**